openghg/util/_user.py

```diff
@@ -153,3 +146,30 @@ def read_local_config() -> Dict:
 
     config: Dict = toml.loads(config_path.read_text())
     return config
+
+
+def check_config() -> None:
+    """Check that the user config file is valid and the paths
+    given in it exist.
+
+    Returns:
+        bool
+    """
+    config_path = get_user_config_path()
+
+    if not config_path.exists():
+        logger.warning("Configuration file does not exist. Please create it by running openghg --quickstart.")
```

Comment:
Does this mean we would now expect the user to always run `$ openghg --quickstart` on the command line?

Reply:
Ah good point, I'll update that so it also asks them to call `create_config`. I'll check the setup documentation as well.